src/pages/Strategies/hooks/use-get-strategy-position.ts

```diff
@@ -0,0 +1,62 @@
+import { CurrencyExt } from '@interlay/interbtc-api';
+import { MonetaryAmount } from '@interlay/monetary-js';
+
+import { useGetAccountPositions } from '@/hooks/api/loans/use-get-account-positions';
+import { CollateralPosition } from '@/types/loans';
+
+import { StrategyType } from '../types';
+import { StrategyData } from './use-get-strategies';
+
+type StrategyPositionData = {
+  amount: MonetaryAmount<CurrencyExt>;
+  earnedAmount?: MonetaryAmount<CurrencyExt>;
+  loanPosition: CollateralPosition;
+};
+
+type UseGetStrategyPositionResult = {
+  isLoading: boolean;
+  data: StrategyPositionData | undefined;
+};
+
+const useGetStrategyPosition = (strategy: StrategyData | undefined): UseGetStrategyPositionResult => {
```

Comment:
It would be much better to pass `StrategyType` here and get currency from statically defined data - to avoid need of being dependent on `getStrategy` from other hook that fetches data.

Reply:
The strategy page is dependent on that data because we need the info about the asset to be able to construct the form and also show the apy. So detaching them would not bring any benefit, at least for now that we do not do any partial loadings of the page (we are yet to implement skeletons)

Reply:
Yes, Strategy page is dependent, but I think it makes sense to minimize dependency of the hook if possible - we might want to use it later also elsewhere where the other dependency doesn't have to happen. Otherwise it was unnecessary to split the hook into separate hooks, no?